visivo/parsers/core_parser.py

```diff
@@ -34,7 +34,7 @@ def __build_project(self):
 
     def __merged_project_data(self):
         project_data = self.project_file_data()
-
+        self.__recursively_add_file_path(project_data, self.project_file)
```

Comment:
I wonder if there is a better way to accomplish this.  It feels brittle.  The issue makes sense, we don't find anything because it is in a folder.  Would it make sense to keep everything working make the change in a different location.  Maybe update the working dir or similar to keep it simple here.